libs/experimental/langchain_experimental/plan_and_execute/agent_executor.py

```diff
@@ -35,6 +36,10 @@ def input_keys(self) -> List[str]:
     def output_keys(self) -> List[str]:
         return [self.output_key]
 
+    @property
+    def intermediate_steps_keys(self) -> List[tuple]:
```

Comment:
thanks for the pr @adrijanik! what's this property used for? (the others `..._keys` properties are required by parent classes)

Reply:
hi @baskaryan, thanks for pointing that out, it has no major function apart from me trying to be consistent with the rest of the `..._keys`.